src/ParseServer.js

```diff
@@ -374,6 +374,16 @@ class ParseServer {
 
 function addParseCloud() {
   const ParseCloud = require('./cloud-code/Parse.Cloud');
+  Object.defineProperty(Parse, 'Server', {
+    get() {
+      return Config.get(Parse.applicationId);
+    },
+    set(newVal) {
```

Comment:
Could it be that the `set` block is not covered by a test?

Reply:
It is covered by `Parse.Server = config`, it's in case the developer wants to mutate config.